packages/avocode-email-tagsinput/test/e2e/avocode-email-tagsinput.spec.js

```diff
@@ -0,0 +1,154 @@
+import puppeteer from 'puppeteer'
+import { expect } from 'chai'
+import {
+  URLRoot,
+  tagsInputFieldSelector,
+  getAnchorAndFocus,
+  addEmailTag,
+  getTagNodes, getInputText,
+} from './test-utils'
+describe('Avocode Email Tags Input', () => {
```

Comment:
1. Please respect meaningful whitespace here. Put new line between imports and describe block, this looks really ugly.
2. Let's keep this consistent with `better-react-tagsinput` library, let's just add name of the component here: `AvocodeEmailTagsinput`

Reply:
Can you fix this please?